cs3/tx/v1beta1/resources.proto

```diff
@@ -0,0 +1,69 @@
+// Copyright 2018-2019 CERN
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// In applying this license, CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+syntax = "proto3";
+
+package cs3.tx.v1beta1;
+
+option csharp_namespace = "Cs3.Tx.V1Beta1";
+option go_package = "txv1beta1";
+option java_multiple_files = true;
+option java_outer_classname = "ResourcesProto";
+option java_package = "com.cs3.tx.v1beta1";
+option objc_class_prefix = "CTX";
+option php_namespace = "Cs3\\Tx\\V1Beta1";
+
+// TxInfo represents information about a transfer.
+message TxInfo {
```

Comment:
Shouldn't we store info about who created the transfer? How do we expect GetTransferStatus to be called? It's unlikely users will remember the transfer ID. Even if they do, they will be able to access transfers created by anybody.

Same for the recipient.

Reply:
I believe it's more like: what do we want to expose in TxInfo. The transfer id would be the bare minimum for transfer functionality so the user can do 'status notification' and 'cancel transfer' requests. It's up to the user/EFSS to keep hold of that transfer id if he wants to use these methods.
All necessary info (transfer id, resource, who initiated, destination, etc.) concerning this request must be persisted in the IOP in a transfer db anyway to make data transfer work. This is also where the linking between transfer id and the resource exists (https://github.com/cs3org/cs3apis/pull/81#discussion_r488507054).
But yes maybe it is 'user friendly' to add some additional info in the TxInfo. What do you think would be helpful there ?